### PR TITLE
Custom routes

### PR DESCRIPTION
**Pull request by nwogu**
#4  Changes made by this pull request
- Add custom route feature to the paystack class
- Add tests for custom route feature
- Update readme to include custom route example

### CODE REVIEW

File: src/Paystack.php [hunk 1]
```diff
@@ -3,12 +3,14 @@
 namespace Yabacon;
 
 use \Yabacon\Paystack\Helpers\Router;
+use Yabacon\Paystack\Contracts\RouteInterface;
```
Comment (ibrahimlawal) [inline src/Paystack.php]:
```suggestion
use \Yabacon\Paystack\Contracts\RouteInterface;
```

File: src/Paystack.php [hunk 2]
```diff
@@ -25,6 +27,25 @@ public function useGuzzle()
         $this->use_guzzle = true;
     }
 
+    public function useRoutes(array $routes)
+    {
+        foreach ($routes as $route => $class) {
+            if (! is_string($route)) {
```
Comment (ibrahimlawal) [inline src/Paystack.php:33]: can we also confirm that it does not shadow an existing route?

**Comment by ibrahimlawal** [comment]
Love that the test coverage improved! Kindly check the review comments @nwogu

**Comment by nwogu** [comment]
> Love that the test coverage improved! Kindly check the review comments @nwogu

Thank you @ibrahimlawal I've seen the comments. I'll update accordingly.

File: src/Paystack.php (hunk 2 above)
Comment (nwogu) [reply src/Paystack.php:33]: Done cc @ibrahimlawal

**Pull request merged**